modules/manage/partials/iceberg/query-iceberg-topics.adoc

```diff
@@ -76,8 +76,7 @@ rpk registry schema create ClickEvent-value --schema path/to/schema.avsc --type
 echo '"key1" {"user_id":2324,"event_type":"BUTTON_CLICK","ts":"2024-11-25T20:23:59.380Z"}' | rpk topic produce ClickEvent --format='%k %v\n' --schema-id=topic
 ----
 +
-The `value_schema_id_prefix` requires that you produce to a topic using the Schema Registry wire format, which includes the magic byte and schema ID in the prefix of the message payload. This allows Redpanda to identify the correct schema version in the Schema Registry for a record. See the https://www.redpanda.com/blog/schema-registry-kafka-streaming#how-does-serialization-work-with-schema-registry-in-kafka[Understanding Apache Kafka Schema Registry^] blog post to learn more.
-
+The `value_schema_id_prefix` mode requires that you produce to a topic using the Schema Registry wire format, which includes the magic byte and schema ID in the prefix of the message payload. This allows Redpanda to identify the correct schema version in the Schema Registry for a record. 
```

Comment:
link to examples like in the modes doc?

Reply:
Added link to new section on wire format